projects/distributed-tracing/src/shared/dashboard/widgets/waterfall/waterfall/waterfall-chart.service.ts

```diff
@@ -89,7 +90,16 @@ export class WaterfallChartService {
           id: node.id,
           start: node.startTime,
           end: node.endTime,
-          color: node.color!
+          color: node.color!,
+          markers: node.logEvents.map((logEvent: LogEvent) => {
+            const timeInMs = this.dateCoercer.coerce(logEvent.timestamp)!.getTime();
```

Comment:
nit: you can make this an inline usage. only one place uses this.

Reply:
Done!!